setup.py

```diff
@@ -23,11 +23,11 @@
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=[
         'd3m',
-        'featuretools @ git+https://github.com/featuretools/featuretools.git@2204f0ee302b69e75e096035e5c628f96b14fb78#egg=featuretools',
+        'featuretools',
```

Comment:
we might need to keep this until we do the release with fixes

Reply:
Will leave the old version for now and we can revisit depending on where we are by the end of the week.